src/Calculator/App.xaml.cs

```diff
@@ -179,234 +122,45 @@ private void OnAppLaunch(IActivatedEventArgs args, string argument)
-                    {
-                        // When the navigation stack isn't restored navigate to the first page,
-                        // configuring the new page by passing required information as a navigation
-                        // parameter
-                        if (!rootFrame.Navigate(typeof(MainPage), argument))
-                        {
-                            // We couldn't navigate to the main page,
-                            // kill the app so we have a good stack to debug
-                            throw new SystemException();
-                        }
-                    }
-                    if (ApplicationView.GetForCurrentView().ViewMode != ApplicationViewMode.CompactOverlay)
-                    {
-                        if (!Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.Phone.UI.Input.HardwareButtons"))
-                        {
-                            // for tablet mode: since system view activation policy is disabled so do ShowAsStandaloneAsync if activationViewSwitcher exists in
-                            // activationArgs
-                            ActivationViewSwitcher activationViewSwitcher = null;
-                            var activateEventArgs = (args as IViewSwitcherProvider);
-                            if (activateEventArgs != null)
-                            {
-                                activationViewSwitcher = activateEventArgs.ViewSwitcher;
-                            }
-                            if (activationViewSwitcher != null)
-                            {
-                                var viewId = (args as IApplicationViewActivatedEventArgs).CurrentlyShownApplicationViewId;
-                                if (viewId != 0)
-                                {
-                                    _ = activationViewSwitcher.ShowAsStandaloneAsync(viewId);
-                                }
-                            }
-                        }
-                        // Ensure the current window is active
-                        Window.Current.Activate();
-                    }
-                }
+            // When the navigation stack isn't restored navigate to the first page,
+            // configuring the new page by passing required information as a navigation
+            // parameter
+            if (rootFrame.Content == null && !rootFrame.Navigate(typeof(MainPage), argument))
+            {
+                // We couldn't navigate to the main page, kill the app so we have a good
+                // stack to debug
+                throw new SystemException("6d430286-eb5d-4f8d-95d2-3d1059552968");
             }
-        }
 
-        private void DismissedEventHandler(SplashScreen sender, object e)
-        {
-            _ = SetupJumpList();
-        }
+            // SetPreferredMinSize should always be called before Window.Activate
+            appView.SetPreferredMinSize(minWindowSize);
```

Comment:
This is also set in line 115. Perhaps we can move it forward.

Reply:
removed the first one.